### ISSUE TITLE
Make .NET MAUI a first-class part of the LTS cycle

### ISSUE THREAD

**Issue by thirstyape** [dotnet/maui#19160]
### Description

For some reason, even though MAUI is supposed to be part of .NET and is on the same release cycle, it is not treated the same as the rest of the framework. While .NET as a whole has LTS releases every 2 years, .NET MAUI does not have any sort of LTS release at all. This means that while developers can rely on having 3 years of stability for their production .NET apps, they can only rely on having 1 year for .NET MAUI apps. 

In a multi-platform world this simply does not make sense. What are we supposed to do here, have part of our production apps running on an LTS and our .NET MAUI apps running on a different version of .NET? In my opinion this completely defeats the purpose of even having an LTS release cycle.

The request is simple, delete the .NET MAUI Support Policy and include .NET MAUI in the standard one.

**References**
[.NET Support Policy](https://dotnet.microsoft.com/en-us/platform/support/policy)
[.NET MAUI Support Policy](https://dotnet.microsoft.com/en-us/platform/support/policy/maui)

### Public API Changes

None, or maybe a ton, the SDK would need to include prior .NET MAUI workloads so that these apps can be built with newer releases.

### Intended Use-Case

Saving developers from the headache of needing to set up multiple installs of VS on multiple machines just to keep production apps running.

**Comment by Vroomer** [comment]
Android and iOS stores force developers to update their apps to target new APIs, which frequently introduce breaking changes. It wouldn't be possible to react to their release cycles in LTS.

**Comment by thirstyape** [comment]
> Android and iOS stores force developers to update their apps to target new APIs, which frequently introduce breaking changes. It wouldn't be possible to react to their release cycles in LTS. You can target multiple .NET versions in VS and MAUI is now served in nugets that can be easily changed.

First, you can no longer target .NET MAUI for .NET 6.0 in VS 17.8 and onwards, which is a rather large issue.

Next, for app updates, the Google Play store still supports API 31, which has been available since late 2021 [source](https://support.google.com/googleplay/android-developer/answer/11926878). For the Apple App Store, it does look like XCode 15 and the iOS 17 SDK will be required soon [source](https://developer.apple.com/ios/submit); however, as VS for Mac has been discontinued and you need to pair VS to a Mac or iOS device to publish [source](https://learn.microsoft.com/en-us/dotnet/maui/ios/deployment/publish-app-store?view=net-maui-8.0&tabs=vs#publish-the-app), this effectively makes the iOS SDK level irrelevant from the .NET MAUI standpoint. Not to mention that you can still build for iOS 12 on XCode 15 [source](https://developer.apple.com/support/xcode).

In summary, I could not disagree more.

**Comment by Vroomer** [comment]
Only existing apps can still target API 31
https://support.google.com/googleplay/android-developer/answer/11926878?hl=en
> Starting August 31, 2023: New apps must target Android 13 (API level 33) or higher

I don't think it's viable to have .NET version that can't be used for new apps. Besides what updates would you except for LTS version, when it can't be updated to target latest API as it would introduce breaking changes?

VS for Mac was essentially replaced by VS Code with MAUI extension.

**Comment by thirstyape** [comment]
This issue is not to request feature updates, rather security updates and the ability to continue to support production applications. There are many cases where a deployed application will have hundreds, thousands, or more users and cannot simply have its target updated to the latest version of .NET right away.

I doubt that many developers are looking to create new .NET MAUI projects with an older version of .NET MAUI, similarly to creating a new .NET project with an old version. This is why I only mentioned app updates.

Personally I already have .NET MAUI apps targeting .NET 8, but I also have others that require maintenance updates and cannot be updated from .NET 6 quite yet.

On the VS Code extension, it has the same requirements as VS, the build will still go through XCode.

**Comment by Redth** [comment]
As @Vroomer pointed out, we are obligated to keep pace with Apple and Google's requirements for submitting apps to their stores.  While I understand that there may be some cases where internal or alternative store distributions are in use, but the majority of our customers end up publishing to these app stores.  We also want to ensure our customers are building apps with updated tooling and SDKs from Apple and Google to ensure they are receiving security updates and fixes in that part of the stack.  Also, in the case of Xcode at least, newer Apple macOS versions no longer continue to work with older versions of Xcode itself which also forces us to continue to support the latest versions of Xcode.

Since we need to continue to support the latest versions of Xcode it's worth pointing out that Xcode is what drives API changes in their own SDK.  Xcode 15 for example introduces breaking API changes (removal of NewsstandKit is one example).  So even if we did want to continue to support net6.0-ios as a LTS, that would mean introducing breaking changes to net6.0-ios's API surface anyway.

Given all of the factors involved here, many of which we have no control over, we've determined our current support policy is the best way to provide value and support to our customers.

**Comment by thirstyape** [comment]
I suppose this would be a design goal difference between Xamarin and MAUI then. That is, having a single date for the support lifecycle instead of per-platform dates. Really too bad in the case of the Window MSIX packages and from the sounds of it Android.
